qa/suites/upgrade/reef-p2p/reef-p2p-stress-split/6-final-workload/rbd-python.yaml

```diff
@@ -0,0 +1,10 @@
+meta:
+- desc: |
+   librbd python api tests
+tasks:
+- workunit:
+    tag: reef
```

Comment:
Hmm. `tag` here as well is it value look confusing. I speculate it's ignored, and thus the comment is just a nit-picker.

Reply:
I think `branch` and `tag` are treated similar by teuthology -- it doesn't attempt to do any validation and simply invokes git.  Based on my reading of `qa/tasks/util/workunit.py`, for a tag it would be something like:
```
$ git clone https://github.com/ceph/ceph.git <path>
$ cd <path>
$ git checkout reef
```
while for a branch something like:
```
$ git clone --depth 1 --branch reef https://github.com/ceph/ceph-ci.git <path>
$ cd <path>
```
So `tag` isn't only confusing but also much less efficient than `branch` here because the entire repo is cloned.  I actually asked to make it `branch: reef` in my review https://github.com/ceph/ceph/pull/55714#discussion_r1501236372.

Reply:
replaced `tag` with `branch`